src/pages/content-edit/head.css

```diff
@@ -1,36 +1,22 @@
 @value pure: "../../_pure.css";
-@value   icons_css: "../../icons.css";
-@value  layout_css: "../layout/layout.css";
+@value icons_css: "../../icons.css";
+@value layout_css: "../layout/layout.css";
 @value buttons_css: "../../buttons.css";
 @value header: "../layout/header.css";
 
-.head {
+.contentHead {
```

Comment:
Code imperialism.

Changed back to `.contentHead`. What do you think about renaming the content body to `.contentBody`? Getting a grasp of the different sections was a little tricky at first.

```
+--------------------------------------------------+
|                   container                      |
| +----------+ +---------------------------------+ |
| |  header  | |             content             | |
| | +------+ | | +-----------------------------+ | |
| | | head | | | |         contentHead         | | |
| | +------+ | | +-----------------------------+ | |
| | +------+ | | +-----------------------------+ | |
| | | body | | | |             body            | | |
| | |      | | | |                             | | |
| | |      | | | |                             | | |
| | |      | | | |                             | | |
| | |      | | | |                             | | |
| | +------+ | | +-----------------------------+ | |
| |          | |                                 | |
| +----------+ +---------------------------------+ |
|                                                  |
+--------------------------------------------------+
```